src/main/java/com/st/utopia/cancellationservice/model/Ticket.java

```diff
@@ -0,0 +1,103 @@
+package com.st.utopia.cancellationservice.model;
+
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+/**
+ * An airport model.
+ *
+ * @author Henry Cho
```

Comment:
In general, move the `@author` tag (and any other Javadoc tags) to the end of the comment block.